src/k8-types/src/store.rs

```diff
@@ -157,7 +157,7 @@ impl MetaItemContext {
 #[macro_export]
 macro_rules! default_store_spec {
     ($spec:ident,$status:ident,$name:expr) => {
-        impl crate::store::StoreSpec for $spec {
+        impl $crate::store::StoreSpec for $spec {
```

Comment:
I don't know enough about rust macros to say if this incorrect or not but it's unclear why it's needed. I'm guessing clippy said so?

Reply:
yes, clippy!